bigquery/unit_tests/test__helpers.py

```diff
@@ -491,6 +491,363 @@ def test_instance_setter_w_bad_udfs(self):
         self.assertEqual(instance.udf_resources, [])
 
 
+class Test_AbstractQueryParameter(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery._helpers import AbstractQueryParameter
```

Comment:
I'd argue against it, myself, and I certainly don't think it makes sense to hold up unrelated PRs to enforce a new and debatable style rule.